src/main/java/at/hannibal2/skyhanni/features/misc/InGameDateDisplay.kt

```diff
@@ -16,13 +18,31 @@ class InGameDateDisplay {
     @SubscribeEvent
     fun onTick(event: LorenzTickEvent) {
         if (!isEnabled()) return
-        if (!event.repeatSeconds(config.RefreshSeconds)) return
+        if (!LorenzUtils.inSkyBlock) return
+        if (!config.useScoreboard && !event.repeatSeconds(config.refreshSeconds)) return
+        if (config.useScoreboard && !event.repeatSeconds(1)) return
 
         checkDate()
     }
 
     private fun checkDate() {
-        display = SkyBlockTime.now().formatted()
+        val date = SkyBlockTime.now()
+        var theBaseString = ""
+        if (config.useScoreboard) {
+            val list = ScoreboardData.sidebarLinesFormatted //we need this to grab the moon/sun symbol
+            val year = "Year ${date.year}"
+            val monthAndDate = list.find{ (it.contains("Winter ") || it.contains("Spring ") || it.contains("Summer ") || it.contains("Autumn ")) && (it.contains("st") || it.contains("nd") || it.contains("rd") || it.contains("th")) } ?: "??"
```

Comment:
Please use mixins for stuff like `list.find{ (it.contains("Winter ") || it.contains("Spring ") || it.contains("Summer ") || it.contains("Autumn "))`

Reply:
hanni what the heck is a mixin

Reply:
I confused mixin with regex in that context. I'm sorry

Reply:
it has been done.